services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersValidateSimpleFlowSpec.groovy

```diff
@@ -12,69 +12,64 @@ import groovy.transform.Memoized
 import org.springframework.beans.factory.annotation.Autowired
 import spock.lang.Narrative
 
-@Narrative("""This test suite check the meter validate request on a flow that contains 2+ switches.
-description of fields:
-- missing - those meters/rules, which is NOT exist on a switch, but it is exist in db
+@Narrative("""This test suite check the meter validate feature on a flow that contains 2+ switches.
+Description of fields:
+- missing - those meters/rules, which are NOT present on a switch, but are present in db
 - misconfigured - those meters which have different value (on a switch and in db) for the same parameter
-- excess - those meters/rules, which is exist on a switch, but it is NOT exist in db
-- proper - meters/rules value is the same on a switch and in db
+- excess - those meters/rules, which are present on a switch, but are NOT present in db
+- proper - meters/rules values are the same on a switch and in db
 """)
 class MetersValidateSimpleFlowSpec extends BaseSpecification {
     @Autowired
     SwitchHelper switchHelper
 
-    def "Able to get meter validate information"() {
+    def "Switch validation is able to return correct information about a flow"() {
         when: "Create a flow"
         def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
-        def defaultMetersSrcSwitch = northbound.getAllMeters(srcSwitch.dpId)
-        def defaultMetersDstSwitch = northbound.getAllMeters(dstSwitch.dpId)
+        def srcSwitchDefaultMeters = northbound.getAllMeters(srcSwitch.dpId)
+        def dstSwitchDefaultMeters = northbound.getAllMeters(dstSwitch.dpId)
         def flow = flowHelper.addFlow(flowHelper.randomFlow(srcSwitch, dstSwitch))
 
-        then: "One meter is automatically created on the both switches"
-        def createdMetersIdSrcSwitch = getCreatedMetersId(srcSwitch.dpId, defaultMetersSrcSwitch)
-        def createdMetersIdDstSwitch = getCreatedMetersId(dstSwitch.dpId, defaultMetersDstSwitch)
-        def burstSizeSrcSwitch = switchHelper.getExpectedBurst(srcSwitch.dpId, flow.maximumBandwidth)
-        def burstSizeDstSwitch = switchHelper.getExpectedBurst(dstSwitch.dpId, flow.maximumBandwidth)
-
-        createdMetersIdSrcSwitch.size() == 1
-        createdMetersIdDstSwitch.size() == 1
+        then: "One meter is automatically created on the src and dst switches"
+        def srcSwitchCreatedMetersId = getCreatedMetersId(srcSwitch.dpId, srcSwitchDefaultMeters)
```

Comment:
If srcSwitchCreatedMetersId is a Collection (List, Array etc.) it should **always** be named in plural. For now it reads like you have gained a certain single 'id' that corresponds to multiple meters. But in real you have a list of id**S**

Reply:
Fixed